airgun/entities/subscription.py

```diff
@@ -160,10 +166,6 @@ class SubscriptionList(SubscriptionNavigationStep):
 
     def pre_navigate(self, _tries, *args, **kwargs):
         super(SubscriptionList, self).pre_navigate(_tries, *args, **kwargs)
-        wait_for(
-                lambda: not self.view.fake_fade_widget.is_displayed,
-                handle_exception=True, logger=self.view.logger, timeout=10 * 60
-        )
```

Comment:
If it's safe to remove this `wait_for`, do we still need `pre_navigate` at all?

Reply:
Yes this safe run the test many time no problem faced, even was not able to find fade elelemt

Reply:
yes right no need for pre_navigate function , thanks to point it